xep-0084.xml

```diff
@@ -37,6 +37,12 @@
   &pgmillard;
   &temas;
   &xvirge;
+  <revision>
+    <version>1.1.3?</version>
```

Comment:
@vanitasvitae Please don’t do that questionmark thing. Git will show me a conflict either way if another version was released inbetween. I might not look closely at the version after initially reviewing the PR, however, unless there’s a conflict.

(And I don’t have a label for "It has a revision block, but I definitely need to pay attention to it beyond the date.)

Reply:
Sorry, I'll patch that out pretty quick :)
For the future: Do you prefer to do the revision block yourselves or what should be filled out by the PR creator?

Reply:
@vanitasvitae I prefer if the PR creator does it, because I don’t always know the preferences regarding initials and remark style. I reserve the right to criticize them though ;)